Nodejs/Tests/AnalysisTests/AnalysisFile.cs

```diff
@@ -131,6 +131,20 @@ static class Analysis {
             List<AnalysisFile> files = new List<AnalysisFile>();
             foreach (var file in Directory.GetFiles(directory, "*", SearchOption.AllDirectories)) {
```

Comment:
The issue with the test as its written now is that it's not actually testing the code that's in the product itself.